### ISSUE TITLE
Meaning of explicit labels

### ISSUE THREAD

**Issue by paulthemagno** [micbuffa/WasabiDataset#1]
I'm doing some experiments with a BERT-like model on Wasabi dataset.
I found out that the proportion between the explicit labels is:
```
0.0      663145
1.0      52342
2.0      445747
3.0      7708
6.0      171837
nan      758775
```

From the [paper](https://arxiv.org/pdf/1912.02477.pdf) I have understood `0.0` is `not explicit `and `1.0` is `explicit`. I know that between other labels there are censored lyrics, but I haven't found out other details online. Can I have an explanation on the meaning of `2.0` , `3.0` and `6.0` ? Can any of them be considered as `explicit`?

Another question is about the dataset used to make evaluations with different models like BERT. I read from the paper 
` In our experiments, we worked with 179k lyrics that carry gold labels provided by Deezer (17k tagged as explicit)`
and after some rows:
`Our corpus contains 52k tracks labelled as explicit and 663k clean (not explicit) tracks17. We have trained a classifier (77.3% f-score on test set) on the 438k English lyrics which are labelled and classified the remaining 455k previously untagged English tracks. `
![image](https://user-images.githubusercontent.com/38130299/92000343-b5605a00-ed3d-11ea-89d1-3f04630f51ae.png)

Were these results obtained from a subset of the WASABI dataset? Is this available online to reproduce the same test?

**Comment by TuringTrain** [comment]
We have not used the explicit labels except from 0 and 1 for the task as we concluded that only 0 and 1 can be safely considered. One of them is censored lyrics, as you correctly said. For details on the others I have to refer you to the original authors as I have these details not available anymore.

The 179k lyrics we used for explcit lyrics detection refers to our paper on said topic (https://www.aclweb.org/anthology/R19-1039.pdf) and is indeed a subset, the one we had available at the time of researching the topic. For reproducibility and comparison I provide the 179k indices here: https://mega.nz/file/aFg3mCjb#Mgt3dDmZlQ6CebOQNAsOcJRelSy1XJRe46ZLsjVW7_0
